dolt_integrations/dvc/examples/fashion-mnist/dolt-dvc-demo.md

```diff
@@ -0,0 +1,610 @@
+are more difficult to categorize, can be seen by comparing
+accuracies between training runs:
+```bash
+> dolt sql -q "with t as (
+  select
+    case
+      when p1.pred = p2.actual then 1
+      else 0
+    end as correct,
+    p1.actual,
+    p1.commit_hash
+  from dolt_history_predictions p1
+  join dolt_history_predictions p2
+    on p1.row_id = p2.row_id and
+       p1.commit_hash = p2.commit_hash
+  )
+  select
+    sum(correct)/count(*),
+    t.commit_hash,
+    count(*) as row_number
+  from t
+  group by commit_hash"
++-----------------------+------------+----------------------------------+
+| sum(correct)/count(*) | row_number | commit_hash                      |
++-----------------------+------------+----------------------------------+
+| 0.6217                | 10000      | vq01vp8o1fsdq95eo4sutmppfhovejfv |
+| 0.5352                | 10000      | foo013e1hgej8oa0282bv1ecb9280mlh |
++-----------------------+------------+----------------------------------+
+```
+
+We use the `dolt_history` table above to access every version of
+the prediction table, compare results, and calculate overall accuracy.
+This query can be broken down into 1) a `with` block that adds a correct
+column that's either 0 or 1, and 2) a `select` statement that performs
+accuracy arithmetic, and 3) a `group by` to bucket accuracy by training
+run.
+Computing every training accuracy requires the history table because predictions are
+replaced every iteration.
+
+We also added a summary row each training run, an easier way to
+incrementally generate logs:
+```
+> > dolt sql -q "select * from summary"
++-----------+--------------+--------------------------------------+--------+
+| loss      | label_branch | timestamp                            | acc    |
++-----------+--------------+--------------------------------------+--------+
+| 1.6994121 | fashion      | 2021-04-02 10:29:12.840018 +0000 UTC | 0.5352 |
+| 1.580542  | master       | 2021-04-02 10:43:14.570946 +0000 UTC | 0.6217 |
++-----------+--------------+--------------------------------------+--------+
+```
+
+One instance where we might want to access the commit history is
+`dolt blame`’ing a data changes. For example, identifying who added
+certain rows to the fashion branch:
+```bash
+dolt blame fashion labels | head -n 5
++--------+----------------------------+-----------------+------------------------------+----------------------------------+
+| ROW_ID | COMMIT MSG                 | AUTHOR          | TIME                         | COMMIT                           |
++--------+----------------------------+-----------------+------------------------------+----------------------------------+
+| 14648  | Add Fashion labels         | Bojack Horseman | Wed Mar 31 16:20:50 PDT 2021 | 5fblpjp5neurvsfp989s6ea9lt01vd2q |
+| 27051  | Add Fashion labels         | Bojack Horseman | Wed Mar 31 16:20:50 PDT 2021 | 5fblpjp5neurvsfp989s6ea9lt01vd2q |
```

Comment:
Maybe do this from SQL?

Reply:
yeah that'd be better, i was a bit clumsy with that table when i first wrote this but i'll take another look